lib/checkuninitvar.cpp

```diff
@@ -1112,6 +1112,9 @@ bool CheckUninitVar::isVariableUsage(const Token *vartok, bool pointer, Alloc al
     if (vartok->strAt(1) == "]")
         return true;
 
+    if (vartok->astParent() && (vartok->astParent()->tokType() == Token::Type::eBitOp))
```

Comment:
as far as I know the "eBitOp" is set using a simple string match so it is set for "address of" operator also. or what is the tokType() return value for an "address of"? maybe you can add a `vartok->astParent()->isBinaryOp()` ..

Reply:
You are right.
I updated implementation and added one more test to check "address of" operator case.